.changes/next-release/api-change-pinpointsmsvoice-82194.json

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "api-change",
+  "category": "``sms-voice``",
+  "description": "Update sms-voice client to latest version."
```

Comment:
I think we should mention something about this service existing for backwards compatibility and people should use `pinpoint-sms-voice` instead.